test/system/src/test/java/io/pravega/test/system/MultiControllerTest.java

```diff
@@ -36,37 +36,35 @@
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.Timeout;
 import org.junit.runner.RunWith;
 import static io.pravega.test.system.framework.Utils.DOCKER_BASED;
 
 @Slf4j
 @RunWith(SystemTestRunner.class)
-public class MultiControllerTest {
-    private static final ScheduledExecutorService EXECUTOR_SERVICE = Executors.newSingleThreadScheduledExecutor();
+public class MultiControllerTest extends AbstractSystemTest {
+
+    @Rule
+    public Timeout globalTimeout = Timeout.seconds(8 * 60);
```

Comment:
The global timeout overrides the timeout mentioned at the start of the test method i.e. @Test(timeout=xxx). Perhaps we should have only one configuration (this pattern is repeated in couple of other tests) ref: https://github.com/junit-team/junit4/wiki/timeout-for-tests

Reply:
Good suggestion, done.